Hamburger ui

Burge337 opened the pull request:
This changes the UI to match what Johannes had proposed in the slack channel. The print and share buttons aren't working yet because this is from staging.
closes #37

Burge337 committed:
Add a menu with links to the viewer page, trash, and user doorboard

Still working on getting the buttons place properly along the toolbar, so as of right now all the elements are aligned to the left. (went a little div crazy here trying out different things, that will probably change)

Also still need to get rid of the links that have been moved: trash, view, print?

The print and share links are not working yet, as I pulled from the staging branch before those features had been implemented.

Progresses issue #37

Burge337 committed:
Put all the elements from the proposed UI changes in place

The print PDF and share link buttons don't work, but all the previous links to trash , viewer page, etc are still working. The UI design was taken from the drawing Johannes submitted in the slack channel.

Closes issue #37

Burge337 committed:
Fix a couple tests breaking because the component didn't have matMenu

Closes #37

floogulinc commented on client/src/app/app.component.html:
```diff
@@ -1,9 +1,40 @@
+    <div class="menu-button-container">
+      <button class="menu-button" mat-button [matMenuTriggerFor]="appMenu">
+        <mat-icon>
+          reorder
```
The `menu` icon would be more appropriate here.

```suggestion
          menu
```

helloworld12321 replied:
Wilco!

helloworld12321 committed:
Merge branch 'staging' into hamburger-ui

helloworld12321 committed:
Use the "menu" icon instead of "reorder"

At Paul's suggestion.

helloworld12321 committed:
Remove the print button

That functionality will probably go inside the share button, when we
implement it.
